react/src/components/masthead/SprkMasthead.stories.js

```diff
@@ -163,6 +163,7 @@ export const defaultStory = () => (
   <SprkMasthead
     littleNavLinks={links}
     narrowNavLinks={links.concat(addedNarrowNavLinks)}
+    narrowNavMenuToggleAriaControls="sprk-masthead-narrow-nav-toggle-aria-default"
```

Comment:
narrowNavId

Reply:
sprk-masthead-narrow-nav-custom-string

Reply:
Lets just remove this, we want them to use the default. Just make sure to have a test for this.